markdown/what.md

```diff
@@ -1,23 +1,79 @@
 # What's this about?
 
 <!-- Note -->
-This is some sample speaker note content. It can contain any Markdown, and
-can span multiple paragraphs.
 
-This is another paragraph. It is separated from another by a blank line.
-However, if a paragraph followed by **two** blank lines creates a new slide:
+I work in Cleura in the education team, and what we do is that we run the learning platforms. Our platforms are based on Open edX, a free open source learning management system.
+Open edX platforms have biannual releases like OpenStack, following a naming convention of botanical tree names. So when I mention the names "Maple", "Lilac" I am reffering to specific releases of the Open edX platform.
+
+Starting from the Maple release of Open edX, the community-supported method of deploying the platform using Ansible playbooks was deprecated. Instead, the upstream decided to transition towards a containerized deployment approach.
+
+To adapt to this change, we made the switch from OpenStack Heat to Kubernetes on OpenStack Magnum. This shift allowed us to embrace containerization for deploying the Open edX platform, leveraging the benefits it provides in terms of scalability, flexibility, and ease of management.
+
+Alongside the deployment changes, we also had a transition in our CI-driven deployment approach. We moved from Gitlab CI to Zuul.
+ 
+This is a summary of the architecture before and after, which also discusses some of the issues we ran into along the way.
 
 
 ## Open edX
 
 <!-- Note -->
-These are the notes for the next slide.
+
+As mentioned earlier, our platform is built on Open edX. Now let me provide you with a concise overview of what Open edX is and its fundamental components.
+
+Harvard University and MIT collaborated to establish edX, a non-profit organization with a mission to promote an open-source vision for the future of education. In 2013, edX made its code open source, enabling universities, corporations, NGOs, government organizations, and entrepreneurs to utilize this powerful tool.
```

Comment:
These two are not sentences you would actually *say*, so please rewrite them to convey the same information in simpler words.